packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -223,6 +225,16 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
     notifyListeners();
   }
 
+  /// The [shape] of the scrollbar's thumb.
```

Comment:
Since this adds a different way to style a scrollbar, we should make sure to thoroughly document what parameters work together and which do not. If I set radius and shape, what happens? If some of these combinations are not allowed, we should assert for them as well.

Reply:
I will see to it the property is thoroughly documented :)

Reply:
Can you remove the self-reference here? If this is documenting `shape`, saying it is `shape` amy not be helpful.

```suggestion
  /// The [OutlinedBorder] of the scrollbar's thumb.
```